module-2/backend/week_one.md

```diff
@@ -1,34 +1,134 @@
-## Week One - Module 2 Recap
+## Week One - Module 2 Recap - Kyle
 
 Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week (which is a TON!). 
 
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  GET: display /read a page
+
+  DELETE: delete from a page / database
+
+  POST: add to a page / database
+
+  PUT: edit a page / database
+
+  HEAD: similar to GET but without the message body (just headers)

```

Comment:
When I think of top five I would usually include PATCH over HEAD, but since it's so similar to PUT I can see why you would leave it out.